### PR TITLE
Include "init-db" in Scanner-DB update steps

### PR DESCRIPTION
**Pull request by RTann**
Version(s):

* RHACS 3.69 - 3.71

Issue:

Link to docs preview:

Additional information:

### CODE REVIEW

File: modules/upgrade-scanner-40.adoc [hunk 1]
```diff
@@ -62,12 +62,7 @@ $ oc -n stackrox set image deploy/scanner scanner=registry.redhat.io/advanced-cl
 +
 [source,terminal,subs=attributes+]
 ----
-$ oc -n stackrox set image deploy/scanner-db db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version}
-----
-+
-[source,terminal,subs=attributes+]
-----
-$ oc -n stackrox set image deploy/scanner-db init-db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version}
+$ oc -n stackrox set image deploy/scanner-db db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version} init-db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version}
```
Comment (RTann) [inline modules/upgrade-scanner-40.adoc:65]: no need for this to be two separate commands, as they can be combined

**Comment by RTann** [comment]
I thought Travis CI would give me a link to a preview?

**Comment by gaurav-nelson** [comment]
Only applies to RHACS 3.71 as RHACS 3.69 and RHACS 3.70 already contain the init-db step as a second step.

**Pull request merged**